datayoga-py/datayoga/blocks/read_csv/block.py

```diff
@@ -12,8 +13,14 @@ class Block(DyBlock):
 
     def init(self, context: Optional[Context] = None):
         logger.debug(f"Initializing {self.get_block_name()}")
+        csv_file = self.properties["file"]
 
-        self.file = self.properties["file"]
+        if os.path.isabs(csv_file) or context is None:
+            self.file = csv_file
+        else:
+            self.file = os.path.join(context.properties.get("job_path"), csv_file)
```

Comment:
this should not be the job path. It should actually be a 'data path' that comes in from the config and has some value that can be overrided somehow if needed.

Reply:
probably a property of the runner

Reply:
data_path is better than job_path, I agree, will change.

Reply:
done